src/packer.pkr.hcl

```diff
@@ -142,13 +151,185 @@ build {
   }
 
   provisioner "powershell" {
-    # Wait 90 seconds before executing the check-defender.ps1 powershell script.
-    # This gives a generous grace period between restarting Windows and running the second provisioner.
-    pause_before = "90s"
-    scripts      = ["src/powershell/check-defender.ps1"]
+    # Wait 30 seconds before executing the next provisioner.
+    # This gives a grace period between restarting Windows and running the next provisioner.
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/initialize-volume.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
+  }
+
+  provisioner "powershell" {
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir D:\\packages"]
+  }
+
+  provisioner "file" {
+    # Upload package lists to the "packages" directory.
+    source      = "src/packages/"
+    destination = "D:\\packages"
+  }
+
+  provisioner "powershell" {
+    # Install general packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=general"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
```

Comment:
Do we really need such a large restart timeout?  Would 5/10/15 minutes not suffice?

Reply:
Yes, considering the longest windows restart took about 26 min
```
2023-08-15T17:02:01Z: ==> amazon-ebs.windows: Restarting Machine
2023-08-15T17:02:01Z: ==> amazon-ebs.windows: Waiting for machine to restart...
2023-08-15T17:02:27Z:     amazon-ebs.windows: EC2AMAZ-E4080NA restarted.
2023-08-15T17:02:27Z: ==> amazon-ebs.windows: Machine successfully restarted, moving on
```

Reply:
I may be losing it but isn't that 26 seconds in your log snippet?

Reply:
@mcdonnnj smh, i'm the one who's losing it, sorry. Yes not sure how I translated that to 26 min. We don't need 30m timeouts after all. pushing a fix.

Reply:
@dav3r @mcdonnnj resolved with [6ae5f16](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/6ae5f166733e60d84ebf13a475d9e147c8a93235)

Reply:
I'm assuming 5 minute time outs would be good enough cushion in case a restart takes longer than usual

Reply:
Sounds good to me.